test/edu-benefits/components/RotcHistoryFields.unit.spec.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+import { expect } from 'chai';
+
+import RotcHistoryFields from '../../../src/js/edu-benefits/components/RotcHistoryFields';
+import { createVeteran } from '../../../src/js/edu-benefits/utils/veteran';
+
+describe('<RotcHistoryFields>', () => {
```

Comment:
I think I've settled on only testing Fields components if they have conditional rendering logic in them. At least for the last few pages I've done. But I don't know what anyone else thinks about that.